Pipfile

```diff
@@ -7,6 +7,9 @@ name = "pypi"
 Django = "==4.0.4"
 psycopg2-binary = "==2.9.3"
 cfenv = "0.5.3"
+pyparsing = "*"
+pydot = "*"
+django-extensions = {git = "https://github.com/django-extensions/django-extensions"}
```

Comment:
We'll want to watch this. Eventually we should switch back to a pip-based install. However, the most recent version of `django-extensions` does not support Django 4. Django 4 support was pushed to main a few days ago, but they haven't published a release yet, so if we install directly from GitHub, we'll get a version that works for us.